directembedding/src/main/scala/ch/epfl/directembedding/DETransformer.scala

```diff
@@ -0,0 +1,77 @@
+package ch.epfl.directembedding
+
+import ch.epfl.directembedding.transformers.{ LiftLiteralTransformation, EndpointTransformation, DSLVirtualization, ReifyAsEmbedding }
+import ch.epfl.yinyang.analysis.FreeIdentAnalysis
+import ch.epfl.yinyang.transformers.{ LanguageVirtualization, NullPreProcessing, NullPostProcessing, PreProcessing, PostProcessing }
+
+import scala.reflect.macros.blackbox
+
+object DETransformer {
+  def apply[C <: blackbox.Context, T](c: C)(
+    _dslName: String,
+    _liftMethodName: String,
+    _virtualizationConfig: c.universe.Type,
```

Comment:
I parse the `Type` into a `Tree` [here](https://github.com/directembedding/directembedding/pull/23/files#r27579658)